frontend/src/Utils/api-discovery/index.test.ts

```diff
@@ -0,0 +1,81 @@
+const cache = {
+  cacheResources: jest.fn(),
+  getCachedResources: jest.fn(),
+};
+
+const resources = {
+  batchResourcesRequest: jest.fn(),
+  getResources: jest.fn().mockReturnValue(Promise.resolve([[]])),
+};
+
+const apiActions = {
+  setResourcesInFlight: jest.fn(),
+  setBatchesInFlight: jest.fn(),
+  receivedResources: jest.fn(),
+};
+
+import { initAPIDiscovery, startAPIDiscovery } from './index';
```

Comment:
NIT: I'm surprised this is allowed by our lint rules, but shouldn't imports appear before our const declarations?